workflows/update_tags.sh

```diff
@@ -75,6 +75,7 @@ function get_latest_tag_for_image() {
     THIS_PREFIX="${THIS_REGISTRY_NAME}/"
     THIS_IMAGE=$(echo "${THIS_IMAGE}" | sed "s#^${DEFAULT_REGISTRY_REGEX}/##")
   fi
+  echo $THIS_PODMAN_TLS $THIS_PREFIX$THIS_IMAGE
```

Comment:
I think this command compromises output from a function, which causes CASMTRIAGE-7016:

    ncn-m001:~ # /usr/share/doc/csm/workflows/update_tags.sh
    Outside loop: registry.local/artifactory.algol60.net/csm-docker/stable/cf-gitea-import
    1.10.0
    Updating tag of artifactory.algol60.net/csm-docker/stable/cf-gitea-import in /usr/share/doc/csm/workflows/1.10.0 to registry.local/artifactory.algol60.net/csm-docker/stable/cf-gitea-import.
    sed: can't read 1.10.0: No such file or directory

Reply:
I believe it also caused [CASMTRIAGE_7088](https://jira-pro.it.hpe.com:8443/browse/CASMTRIAGE-7088)

Reply:
I'm going to open a pull request removing this echo, which does indeed cause a bug as mentioned by @mtupitsyn  above.  In the future, please be sure to test such changes, and please include information in your commit message about why such messages are needed. As it stands, I cannot determine why this `echo` statement was deemed necessary at all.